src/pages/diagram/dialogs/diagram-confirmation/diagram-confirmation.tsx

```diff
@@ -18,7 +17,7 @@ export const DiagramConfirmation: React.FC<Props> = ({ isOpen, onClose }) => {
 
   function onSaveNewDiagram() {
     if (saveConfirmationDialog.isVisible) {
-      dispatch(setSaveDialog({ isVisible: true }));
+      dispatch(setEditDialog({ isVisible: true }));

```

Comment:
Como o component servirá tanto para salvar quanto para editar, trocar o nome da função. (Pode ser o mesmo padrão do nome escolhido para o component)

```suggestion
      dispatch(setFormDialog({ isVisible: true }));

```